### ISSUE TITLE
🐛 Bug: Incorrect sorting of Cyrillic data

### ISSUE THREAD

**Issue by lynx-work** [nocodb/nocodb#8331]
### Please confirm if bug report does NOT exist already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Incorrect sorting of data in Cyrillic.

If you enter data into the table in Cyrillic and then try to sort in alphabetical order, the sorting is incorrect.

### Desired Behavior

Sorting in Cyrillic should be in accordance with the Cyrillic alphabet.
АБВГДЕЁЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯ

### Project Details

Node: **v20.12.1**
Arch: **x64**
Platform: **linux**
Docker: **false**
RootDB: **mysql2**
PackageVersion: **0.205.1**

### Attachments

<img width="607" alt="Снимок экрана 2024-04-24 в 12 04 15" src="https://github.com/nocodb/nocodb/assets/115034853/68293153-5a88-459b-b33d-e019a10abe8d">

**Comment by dstala** [comment]
Works fine with PG DB

**Comment by dstala** [comment]
## Input
Иван (Ivan)
Наталья (Natalya)
Сергей (Sergey)
Мария (Mariya)
Михаил (Mikhail)
Ольга (Olga)
Дмитрий (Dmitriy)
Анастасия (Anastasiya)
Александр (Aleksandr)
Екатерина (Ekaterina)

## Sorted output
Александр (Aleksandr)
Анастасия (Anastasiya)
Дмитрий (Dmitriy)
Екатерина (Ekaterina)
Иван (Ivan)
Мария (Mariya)
Михаил (Mikhail)
Наталья (Natalya)
Ольга (Olga)
Сергей (Sergey)

**Comment by lynx-work** [comment]
Yeah, it's a weird bug. I made a copy of the table and everything sorts normally in it. But the sorting in the original table still does not work properly.
In my configuration using MariaDB as MySQL.

**Comment by dstala** [comment]
closing this for now. if it happens again on latest version, please reopen this ticket
